includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -432,12 +432,13 @@ public function get_fraud_outcome_transactions_export( $request ) {
 	 * @param string $user_email The email to search for.
 	 * @param string $deposit_id The deposit to filter on.
 	 * @param string $locale     Site locale.
+	 * @param string $download_type The type of download to perform.
 	 *
```

Comment:
💭 It may be worth listing the possible values to clarify what this param is for.
```suggestion
	 * @param string $download_type The type of download to perform: 'sync' or 'async'.
```